src/components/molecules/CardInput/CardInput.scss

```diff
@@ -1,28 +0,0 @@
-@import "scss/constants.scss";
-
-.StripeElement {
```

Comment:
Have we searched to ensure that this class (or any of the others in this file) is no longer referenced anywhere in the remaining app code? If so, please say so, then feel free to mark this comment as resolved.